src/member.cpp

```diff
@@ -111,7 +119,7 @@ namespace hypha
 
         Document paymentReceipt(getContract(), getContract(), Payer::defaultReceipt(getAccount(), genesis_voice, memo));
 
-        Edge::write(getContract(), getAccount(), getHash(), paymentReceipt.getHash(), common::PAYMENT);
+        Edge::write(getContract(), getAccount(), primary_key(), paymentReceipt.primary_key(), common::PAYMENT);
```

Comment:
Right, initially I was using primary_key, but then I decided to use getID function since it was clearer than that but forgot to change some of these :p